remme/tp/consensus_account.py

```diff
@@ -145,9 +146,9 @@ def _send_reward(self, context, public_key, pb_payload):
             ZERO_ADDRESS: zero_account,
         }
 
-        if unfreeze_bonus <= reputational < min_stake * unfreeze_bonus:
-            unfrozen_share = self._calculate_share(max_share, min_share, unfreeze_bonus,
-                                                   min_stake, reputational)
+        if UNFREEZE_BONUS <= reputational < min_stake * UNFREEZE_BONUS:
```

Comment:
Here are a mistake and need o change from UNFREEZE_BONUS to min_stake (anyway it will be in another PR)